docs/_docs/api.json

```diff
@@ -1174,6 +1246,7 @@
         }
       }
     },
+    "findResponse": {},
```

Comment:
It doesn't look like findResponse got documented. I would've expected it to match entryList.

Reply:
Huh, I guess `type findResponse entryList` didn't work. Thanks, updated.